user.rb

```diff
@@ -22,20 +26,26 @@ def self.create(first_name:, last_name:, email:)
   end
 
   def self.write_new_user_to_db
```

Comment:
Would this be better named as `write_all_users_to_db`?

Reply:
It certainly would. I originally had two separate methods, but collapsed them into one without updating the name to reflect that.